lib/Jsrt/Jsrt.cpp

```diff
@@ -2746,3 +2762,47 @@ CHAKRA_API JsRunSerializedScriptWithCallback(_In_ JsSerializedScriptLoadSourceCa
     return RunSerializedScriptCore<JsSerializedScriptLoadSourceCallback, JsSerializedScriptUnloadCallback>(nullptr, scriptLoadCallback, scriptUnloadCallback, buffer, sourceContext, sourceUrl, false, result);
 }
 #endif // _WIN32
+
+
+CHAKRA_API JsRunScriptUtf8(
```

Comment:
Since these are new APIs shouldn't we take length of source as well so that we can allow embedded nulls in script. I understand that these APIs are just equivalent of win32 wchar_t versions but these are any how new APIs and we should be careful in what is the final shape otherwise we will have to write new set of APIs for embedded null scenarios in future.